proposals/TEMPLATE.md

```diff
@@ -45,7 +45,7 @@ Explain why you have chosen this rating
 What awesome potential impact/outcomes/results will we see if we nail this project?
 -->
 
-#### Leverage
+#### Internal leverage
```

Comment:
Maybe "Dev Velocity"? internal leverage is jargon to me...

Reply:
Hmmm... this is about `Acceleration`?

Reply:
Agree with @raulk 's interpretation and desire to change this. It's not only about velocity though. Learning and information is an equally or more important aspect. I believe that product-market fit canon emphasises optimising for rate of learning, especially in early stages.

I would actually be happy to change this to "Learning", given that velocity might be captured in Impact.

Or how about "Learning and velocity"?

Reply:
The counter argument here is to just remove this section entirely in favor of impact. Right now impact is aimed at being solely oriented towards PMF and users, but projects can also have a high impact by getting us more information or enabling us to move more quickly. I don't quite get the need to separate the two, my assumption is that we're trying to be able to quickly clarify what has direct and indirect impact. However, with the removal of quantification, https://github.com/protocol/web3-dev-team/pull/20, this feels like an unnecessary distinction and I recommend we simplify to just having an `Impact` section.